disnake/audit_logs.py

```diff
@@ -236,6 +243,8 @@ def _transform_type(
     elif action_name.startswith("integration_") or action_name.startswith("overwrite_"):
         # integration: str, overwrite: int
         return data
+    elif action_name.startswith("onboarding_question_"):
```

Comment:
```suggestion
    elif action_name.startswith("onboarding_prompt_"):
```
see other comment about renaming

Reply:
3f4be9542c46304f1a2d515b6874af8c58364a2a